installation/env

```diff
@@ -0,0 +1,28 @@
+# Go related variables, shouldn't need to be changed
+export GOPATH=$HOME/asic_q2
```

Comment:
asic_q2 is not representative to the project. Can you change it to ztp_go or something similar please?. Thank you

Reply:
will do